services/user.py

```diff
@@ -0,0 +1,37 @@
+from db.models import User
+
+
+def create_user(username: str, password: str,
+                email: str = None, first_name: str = None,
+                last_name: str = None) -> User:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(pk=user_id)
+
+
+def update_user(user_id: int, username: str = None, password: str = None,
+                email: str = None, first_name: str = None,
+                last_name: str = None) -> User:
+    user = User.objects.get(id=user_id)
```

Comment:
Similar to the `get_user` function, consider handling the case where the `User` does not exist by catching the `User.DoesNotExist` exception.